src/test/java/pl/byczazagroda/trackexpensesappbackend/service/WalletServiceImplTest.java

```diff
@@ -64,7 +72,115 @@ void itShouldThrowWhenWalletNotFound() {
         UpdateWalletDTO updateWalletDto = new UpdateWalletDTO(1L, "walletName");
         //WHEN
         //THEN
-        assertThatThrownBy(() -> underTest.updateWallet(updateWalletDto))
+        assertThatThrownBy(() -> walletService.updateWallet(updateWalletDto))
                 .isInstanceOf(ResourceNotFoundException.class);
     }
-}
\ No newline at end of file
+
+    @Test
+    void shouldCreateWalletProperly() {
+//given
```

Comment:
Please do comments formatting in same way for:
// given
// when
// then

Reply:
Formatting of comments refactored.